site/src/components/DeploySettingsLayout/Option.tsx

```diff
@@ -72,18 +82,38 @@ export const OptionValue: FC<OptionValueProps> = (props) => {
     return <span css={optionStyles}>{children}</span>;
   }
 
+  if (typeof children === "object" && !Array.isArray(children)) {
+    return (
+      <ul css={listStyles}>
+        {Object.entries(children).map(([option, isEnabled]) => (
+          <li key={option} css={optionStyles}>
+            <Box
+              sx={{
+                display: "inline-flex",
+                alignItems: "center",
+              }}
+            >
+              {option}
+              {isEnabled && (
+                <CheckCircleOutlined
+                  sx={{
+                    width: 16,
+                    height: 16,
+                    color: (theme) => theme.palette.success.light,
+                    margin: (theme) => theme.spacing(0, 1),
+                  }}
+                />
```

Comment:
I like the check. If nothing is set though, it is tricky to know if nothing is set, or they are all set.

Can we do something to indicate the value is _available_, but not set? Like maybe greying out the words? Just one idea.

![Screenshot from 2023-10-16 10-57-20](https://github.com/coder/coder/assets/5446298/91ffb004-90c5-4f62-b680-67779286776b)

Reply:
Yeah! I like the design you've added above. I'll throw both in Slack and see what folks think.

Reply:
![Screenshot 2023-10-16 at 9 57 13 PM](https://github.com/coder/coder/assets/19142439/4f9d1bbf-a2b4-4709-a7c3-cd4e5dab1396)

I actually kind of like either what I had previously, or what you suggested, but maybe not the mix of both as suggested in Slack.